### PR TITLE
.

### PR DESCRIPTION
**Pull request by shubhamsharele**

### CODE REVIEW

**Comment by palnabarun** [comment]
@shubhamsharele Can you elaborate on the PR please?

File: README.md [hunk 1]
```diff
@@ -1 +1,76 @@
 # Universal-Manipulator
+# Abstract
+
+A mechanical arm, programmable with similar functions to a human arm having. The gripper attached can be used to pick objects from one  given coordinates and drop to another given coordinates .
+
+# Team Members
+<li>Vipin Kumar</li>
```
Comment (palnabarun) [inline README.md]: Use Markdown syntax for all the markup. Remove HTML syntax.

File: README.md [hunk 2]
```diff
@@ -1 +1,76 @@
...
+  
+<p><div><img src="https://github.com/shubhamsharele/Universal-Manipulator/blob/master/Photos/base.png"></div></p>  
+  
+ <li>Base gear</li>
+  
+<p><div><img src="https://github.com/shubhamsharele/Universal-Manipulator/blob/master/Photos/gear.png"></div></p>  
+ 
+ <li>3D printed arms</li>
+
+<p><div><img src="https://github.com/shubhamsharele/Universal-Manipulator/blob/master/Photos/arm.png"></div></p>
```
Comment (palnabarun) [inline README.md]: Render images using Markdown.

**Review by palnabarun — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -1 +1,129 @@
 # Universal-Manipulator
+# Abstract
+
+A mechanical arm, programmable with similar functions to a human arm having. The gripper attached can be used to pick objects from one  given coordinates and drop to another given coordinates .
+
+# Team Members
+* Vipin Kumar
```
Comment (agshubhs) [inline README.md]: @shubhamsharele write the names in alphabetical order

Comment (agshubhs) [reply README.md]: and also add their github link in it

**Comment by agshubhs** [comment]
@shubhamsharele 
Overall is good make the necessary changes and we will merge your PR.

File: README.md (hunk 3 above)
Comment (shubhamsharele) [reply README.md]: I have made the necessary changes and none of my group members has a GitHub profile

**Comment by agshubhs** [comment]
@shubhamsharele  Nice work

**Pull request merged**